_releases/9.0.0.md

```diff
@@ -9,7 +9,7 @@ summary: >
 
 artifact-root: "https://archive.apache.org/dist/incubator/nuttx/9.0.0"
 checksum-root: "https://archive.apache.org/dist/incubator/nuttx/9.0.0"
-key-file: "https://downloads.apache.org/incubator/nuttx/KEYS"
+key-file: "https://www.apache.org/dyn/closer.lua/incubator/nuttx/KEYS"
```

Comment:
Keys and signatures also _must_ be linked directly to downloads and not via the lua script see https://infra.apache.org/release-download-pages.html

Reply:
Done.